joomla/joomla-cms
# [installation] Clean assets rules in sql files

### Summary of Changes

This PR cleans all empty rules in the assets in tables in all sql installation files.
### Testing Instructions
1. Code review
2. Apply patch, delete configuration.php file, refresh page and install joomla
3. Check joomla install files and ACL works fine
### Documentation Changes Required

None.
